projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.service.ts

```diff
@@ -70,31 +65,34 @@ export class MerchandisingCarouselComponentService {
     return metadata;
   }
 
-  private mapMerchandisingProductsToCarouselItems(
-    merchandisingProducts: MerchandisingProducts
+  private mapStrategyProductsToCarouselItems(
+    strategyProducts: StrategyProducts
   ): Observable<MerchandisingProduct>[] {
-    return merchandisingProducts && merchandisingProducts.products
-      ? merchandisingProducts.products.map((product, index) => {
-          product.metadata = this.getCarouselItemMetadata(product, index + 1);
-          return of(product);
-        })
+    return strategyProducts && strategyProducts.products
+      ? strategyProducts.products.map((strategyProduct, index) =>
+          this.productService.get(strategyProduct.id).pipe(
```

Comment:
From yesterday, we introduced scopes for products. The scopes help reduce the payload, and we can use the `list` scope in this case, which loads the data required to display products in a list view.

To use it, just add a second parameter to `this.productService.get(strategyProduct.id, this.PRODUCT_SCOPE)`.
Define `  protected readonly PRODUCT_SCOPE = 'list';` in the class above the constructor.

Reply:
:+1: added